.github/workflows/build.yml

```diff
@@ -0,0 +1,132 @@
+on:
+  push:
+    tags:
+    - '*'
+    branches:
+    - '*'
+  pull_request:
+    branches:
+    - '*'
+  # repository_dispatch is a newer github-actions feature that will allow building from triggers other than code merge/PR
+  repository_dispatch:
+    types: [build]
+
+name: Build EmuFlight 1
+jobs:
+  build:
+    timeout-minutes: 75
+    strategy:
+      max-parallel: 3
+      matrix:
+        targets: [unified, targets-f4, targets-f7]
```

Comment:
this should be changed to just unified and targets-h7

Reply:
ok cool, thanks for working on this, i suppose we will have to manually add the h7 targets for now or make a rule for building them